### PR TITLE
Implement mission section for Homepage Redesign

### PR DESCRIPTION
**Pull request by pmanal1**
Partially completes [12302](https://app.shortcut.com/clarkcan/story/12302/implement-hero-splash-and-mission-sections).

In this PR:
- Adds mobile and desktop views for the mission section.

## Screenshots

Desktop View
Screen Shot 2022-07-27 at 10 20 06 AM

Mobile View
mission mobile gif

### CODE REVIEW

**Review by EVogel1999 — APPROVED** [review]

File: src/app/cube/new-home/mission/mission.component.html [hunk 1]
```diff
@@ -1 +1,60 @@
...
+        <div class="second-element">
+            <span class="fa-stack fa-2x">
+                <i class="fas fa-circle fa-stack-2x"></i>
+                <i class="far fa-cloud fa-stack-1x"></i>
+            </span>
+            <h2>Easy In / Easy Out</h2>
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                Cursus euismod quis viverra nibh cras pulvinar.
```
Comment (pzalep1) [inline src/app/cube/new-home/mission/mission.component.html:29]: I will get you copy for this as well.

File: src/app/cube/new-home/mission/mission.component.html [hunk 2]
```diff
@@ -1 +1,60 @@
...
+            <h2>Quality Curriculum</h2>
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                Cursus euismod quis viverra nibh cras pulvinar.
+            </p>
+        </div>
+        
+        <div class="second-element">
+            <span class="fa-stack fa-2x">
```
Comment (pzalep1) [inline src/app/cube/new-home/mission/mission.component.html:47]: Could change this class to be more descriptive?

**Review by pzalep1 — COMMENTED** [review]
I will get you copy for this section as well.

**Review by mfranz1 — CHANGES_REQUESTED** [review]
COPY: https://docs.google.com/document/d/1bmTaQ-KjJc1kfdiy9Bh8EFWHQ6mUtSK7kh526YFpvjI/edit?usp=sharing

**Comment by pmanal1** [comment]
### Screenshot of new content text
Screen Shot 2022-08-02 at 2 04 45 PM

**Review by mfranz1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**